py/desisim/lya_spectra.py

```diff
@@ -27,6 +35,9 @@ def get_spectra(lyafile, nqso=None, wave=None, templateid=None, normfilter='sdss
           SEED.
         qso (desisim.templates.QSO, optional): object with which to generate
           individual spectra/templates.
+        add_dlas (bool): Inject damped Lya systems into the Lya forest
+          These are done according to the current best estimates for the incidence dN/dz
```

Comment:
Could you add a citation for "current best estimates" and guidance somewhere of what should be changed if we wanted to update this?

For consideration: From the name of `fN_spline_z24.fits.gz` it sounds like you may be storing spline parameters, which may be a pain to understand/update if we want to update in the future (splines seem straightforward but every implementation seems to parameterize them slightly differently...).  Should we just store the raw histogram numbers to which a spline is fit on the fly?